grid/recruitment/create-lead/profile.supr

```diff
@@ -0,0 +1,443 @@
+"""
+Manage Leads
+Create sourced leads in ATS (Applicant Tracking System).
+"""
+name = "recruitment/create-lead"
+version = "1.0.0"
+
+"""
+Create Lead
+Creates a possible candidate either for some specific job or just for ATS.
+"""
+usecase CreateLead {
+
+  input {
```

Comment:
suggestion: There are many duplicities in the input fields and `Lead` model fields. Consider using Comlink fields to have the fields documented just once in the profile